core/src/test/java/it/pagopa/selfcare/mscore/core/OnboardingServiceImplTest.java

```diff
@@ -225,6 +232,7 @@ void persistOnboarding_whenUserNotExistsOnRegistry() {
         token.setStatus(onboarding.getStatus());
         token.setContractSigned(onboarding.getContract());
 
+        when(pecNotificationConnector.insertPecNotification(any(PecNotification.class))).thenReturn(true);
```

Comment:
you could add verify that PecNotification has expected value using ArgumentCaptor...for ex: 
```
ArgumentCaptor< PecNotification > argCaptor = ArgumentCaptor.forClass(PecNotification.class);
 verify(pecNotificationConnector, times(1)). insertPecNotification(argCaptor.capture());
assertEquals(...., argCaptor.getValue()...)
```